### DISCUSSION TITLE
not getting metamask connected in android using wallet connect v2

### DISCUSSION THREAD

**Discussion by SumiBisht** [WalletConnect/walletconnect-monorepo#3381] [Developer Support]
Using the latest packages but not getting connected to meta mask in mobile and earlier it was getting connected but not anymore it opens the metamask app but no request on the app.
<img width="355" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/40060905/ea562e8e-9a8a-4792-8510-eee843af4d0e">
@glitch-txs

**Comment by glitch-txs** [comment]
Hi, could you provider a minimal reproducible example of the issue?
Also, are you able to connect on labs? 
https://lab-walletconnect-modal.pages.dev/with-ethereum-provider

**Reply by shaheer2020** [reply]
Same issue here. I am unable to get the connect message within the Metamask app. v1 was working fine for me, but this is not working any more now.

**Reply by glitch-txs** [reply]
@shaheer2020 v1 was shut down, consider migrating to v2 :)
https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/overview

**Comment by sachin-kiwi** [comment]
I am also facing same in my personal project when i used by android metamask sometimes but if i use test dapp for connection there is no issue

**Comment by osamajatt** [comment]
I'm encountering a similar problem. When attempting to connect Metamask using WalletConnect, it initiates the Metamask app, but doesn't redirect back to the platform. Moreover, the connection with the platform isn't established; it appears to only enable logging into the Metamask app.

**Comment by Ruithlzz09** [comment]
i found the issue happening due to the localStorage
running command on the browser console
localStorage.clear()

and also clear existing  session from metamask

And it now does not give me any problem
